meson.build

```diff
@@ -0,0 +1,179 @@
+    [
+        'selective_route_exchange',
+        [
+            SRC_DIR / 'crossover' / 'selective_route_exchange.cpp',
+            SRC_DIR / 'crossover' / 'crossover.cpp',
+            SRC_DIR / 'Individual.cpp',
+            SRC_DIR / 'ProblemData.cpp',
+            SRC_DIR / 'PenaltyManager.cpp',
+            SRC_DIR / 'XorShift128.cpp', 
+        ],
+        INSTALL_DIR / 'crossover'
+    ],
+    [
+        'broken_pairs_distance',
+        [
+            SRC_DIR / 'diversity' / 'broken_pairs_distance.cpp',
+            SRC_DIR / 'Individual.cpp',
+            SRC_DIR / 'ProblemData.cpp',
+            SRC_DIR / 'PenaltyManager.cpp',
+            SRC_DIR / 'XorShift128.cpp',
+        ],
+        INSTALL_DIR / 'diversity'
+    ],
+    [
+        'LocalSearch',
+        [
+            SRC_DIR / 'educate' / 'LocalSearch.cpp',
+            SRC_DIR / 'educate' / 'Route.cpp',
+            SRC_DIR / 'educate' / 'Node.cpp',
+            SRC_DIR / 'ProblemData.cpp',
+            SRC_DIR / 'Individual.cpp',
+            SRC_DIR / 'PenaltyManager.cpp',
+            SRC_DIR / 'XorShift128.cpp',
+        ],
+        INSTALL_DIR / 'educate'
+    ],
+    [
+        'Exchange',
+        [
+            SRC_DIR / 'educate' / 'Exchange.cpp',
+            SRC_DIR / 'educate' / 'Node.cpp',
+            SRC_DIR / 'ProblemData.cpp',
+            SRC_DIR / 'PenaltyManager.cpp',
+        ],
+        INSTALL_DIR / 'educate'
+    ],
+    [
+        'MoveTwoClientsReversed',
+        [
+            SRC_DIR / 'educate' / 'MoveTwoClientsReversed.cpp',
+            SRC_DIR / 'educate' / 'Node.cpp',
+            SRC_DIR / 'ProblemData.cpp',
+            SRC_DIR / 'PenaltyManager.cpp',
+        ],
+        INSTALL_DIR / 'educate'
+    ],
+    [
+        'TwoOpt',
+        [
+            SRC_DIR / 'educate' / 'TwoOpt.cpp',
+            SRC_DIR / 'educate' / 'Node.cpp',
+            SRC_DIR / 'ProblemData.cpp',
+            SRC_DIR / 'PenaltyManager.cpp',
+        ],
+        INSTALL_DIR / 'educate'
+    ],
+    [
+        'RelocateStar',
+        [
+            SRC_DIR / 'educate' / 'RelocateStar.cpp',
+            SRC_DIR / 'educate' / 'Exchange.cpp',
+            SRC_DIR / 'educate' / 'Route.cpp',
+            SRC_DIR / 'educate' / 'Node.cpp',
+            SRC_DIR / 'ProblemData.cpp',
+            SRC_DIR / 'PenaltyManager.cpp',
+        ],
+        INSTALL_DIR / 'educate'
+    ],
+    [
+        'SwapStar',
+        [
+            SRC_DIR / 'educate' / 'SwapStar.cpp',
+            SRC_DIR / 'educate' / 'Route.cpp',
+            SRC_DIR / 'educate' / 'Node.cpp',
+            SRC_DIR / 'ProblemData.cpp',
+            SRC_DIR / 'PenaltyManager.cpp',
+        ],
+        INSTALL_DIR / 'educate'
+    ]
+]
+
+foreach extension : extensions
+    message('Going to build ' + extension[0])
+
+    py.extension_module(
+        extension[0],
+        extension[1],
+        dependencies : [py.dependency(), pybind11],
+        install : true,
+        install_dir: extension[2],
```

Comment:
... and here, you could use `subdir: 'pyvrp/crossover'` or similar, and Meson would install to the configured platlibdir override, as '{platlibdir}/pyvrp/crossover/'.